python/manylinux1/build_arrow.sh

```diff
@@ -122,6 +122,9 @@ for PYTHON_TUPLE in ${PYTHON_VERSIONS}; do
     PATH="$PATH:${CPYTHON_PATH}/bin" $PYTHON_INTERPRETER -c "import pyarrow.parquet"
     PATH="$PATH:${CPYTHON_PATH}/bin" $PYTHON_INTERPRETER -c "import pyarrow.plasma"
 
+    echo "=== (${PYTHON}) Install modules required for testing ==="
+    pip install -r requirements-test.txt
```

Comment:
@xhochy please review this

Reply:
I'm a but confused that we install requirements here. They should already be all installed in https://github.com/apache/arrow/blob/0729cb771bd51f60423b52d44a50bddc45653d90/python/manylinux1/scripts/build_virtualenvs.sh#L42-L44

Reply:
It was complaining about missing futures